Embd_Chatbot_adithya/chatbot/README.md

```diff
@@ -12,6 +12,66 @@ This folder contains a standalone AI-powered chatbot interface designed for medi
 - **Responsive Design**: Modern, mobile-friendly interface
 - **Quick Actions**: Pre-configured buttons for common medical tasks
 
+## Quick Start
+
```

Comment:
If you're bringing up a framework, you should just use vite to help build your folder structure